tests/integration/features/steps/integration_steps.py

```diff
@@ -1323,6 +1324,52 @@ def _i_modify_a_statistics_db_file(context, table, keyspace):
         file.write('Adding some additional characters')
 
 
+@then(r'checking the list of decommissioned nodes returns "{expected_node}"')
+def _checking_list_of_decommissioned_nodes(context, expected_node):
+    # Get all nodes having backups
+    storage = Storage(config=context.medusa_config.storage)
+    blobs = storage.list_root_blobs()
+    all_nodes = medusa.purge_decommissioned.get_all_nodes(blobs)
+
+    # Get live nodes
+    live_nodes = medusa.purge_decommissioned.get_live_nodes(context.medusa_config)
+
+    # Get decommissioned nodes
+    decommissioned_nodes = medusa.purge_decommissioned.get_decommissioned_nodes(all_nodes, live_nodes)
+
+    assert expected_node in decommissioned_nodes
+
+
+@when(r'I run a purge on decommissioned nodes')
+def _run_purge_on_decommissioned_nodes(context):
```

Comment:
**Issue:** The logic of `purge_decommissioned.main()` is duplicated here instead of being invoked. This means that any change to the original method will have to be duplicated here, and that the runtime method isn't tested.
Could you replace all this code with just invoking `purge_decommissioned.main()` ?

Reply:
okay